lib/index.js

```diff
@@ -17,7 +17,7 @@ const Tap = require('./tap');
 // Declare internals
 
 const internals = {
-    jsonRegex: /^application\/[a-z.+-]*json$/,
+    jsonRegex: /^application\/[a-z0-9.+-]*json$/,
```

Comment:
Maybe 
`/^application\/([a-z0-9.]*[+-]json|json)$/`
to avoid `application/blahjson` returning true? Likely an edge case, but seems reasonable to prevent